docs/en/connector-v2/sink/Assert.md

```diff
@@ -21,7 +21,7 @@ A flink sink plugin which can assert illegal data by user defined rules
 |rules.field_value              | ConfigList  | no       | -             |
 |rules.field_value.rule_type    | string      | no       | -             |
 |rules.field_value.rule_value   | double      | no       | -             |
-
+| common-options                | string      | no       | -             |
```

Comment:
Type of `common-options` maybe not `string`, I think common-options have no type.

Reply:
The common-options used by the connector V1 version is string, do we need to change it to no?